package.json

```diff
@@ -65,38 +64,34 @@
   },
   "peerDependencies": {
     "mapbox-gl": "0.48.0",
-    "prop-types": "^15.5.10",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1"
   },
   "devDependencies": {
     "@types/core-js": "0.9.43",
     "@types/deep-equal": "1.0.1",
-    "@types/enzyme": "2.8.8",
+    "@types/enzyme": "3.1.13",
     "@types/geojson": "^7946.0.4",
-    "@types/jest": "21.1.0",
+    "@types/jest": "^23.3.1",
     "@types/mapbox-gl": "^0.47.1",
-    "@types/node": "8.0.29",
+    "@types/node": "^10.9.2",
```

Comment:
I believe the `types/node` major versions correspond to Node's major releases, so this is actually a significant change – I think we should revert it for now. Node 10 becomes LTS in October, should revisit the version then.